### DISCUSSION TITLE
A11Y Prefers reduced motion on animations

### DISCUSSION THREAD

**Discussion by janekkkk** [saadeghi/daisyui#2100] [Ideas / feature requests]
Not everyone likes decorative animations or transitions, and some users outright experience motion sickness when faced with parallax scrolling, zooming effects, and so on. The user preference media query prefers-reduced-motion lets you design a motion-reduced variant of your site for users who have expressed this preference.

- https://tailwindcss.com/docs/animation#prefers-reduced-motion 
- https://web.dev/prefers-reduced-motion/

**Comment by saadeghi** [comment]
Thank you.  
I just disabled most of the animations of the homepage for `prefers-reduced-motion`  
2 of them are going to stay because the are working with scroll position and disabling them would result nothing to happen when page is scrolling and that's not good.  

I think it's much better now for people who have `prefers-reduced-motion` enabled.

**Reply by janekkkk** [reply]
I was thinking more to bake this into the DaisyUI components, so users don't have to manually build this into the DaisyUI components.
